src/main/scala/nl/codestar/sbtfindsecbugs/FindSecBugs.scala

```diff
@@ -7,12 +7,6 @@ import Keys._
 
 object FindSecBugs extends AutoPlugin {
   val findsecbugsConfig = sbt.config("findsecbugs")
-  ivyConfigurations += findsecbugsConfig
-  libraryDependencies ++= Seq(
```

Comment:
I wonder, now that these dependencies are also in the plugin's build.sbt, are they still also needed here?